crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -172,8 +172,8 @@ impl FontAtlasSet {
                         .get_glyph_index(cache_key)
                         .map(|location| GlyphAtlasInfo {
                             location,
-                            texture_atlas: atlas.texture_atlas.clone_weak(),
-                            texture: atlas.texture.clone_weak(),
+                            texture_atlas: atlas.texture_atlas.clone(),
```

Comment:
I think these should be switched to AssetIds, as this is just "cache data". The FontAtlas "owns" this handle. The liveness of the atlas shouldn't be tied to this metadata, which may or may not get cleaned up at the same time as the FontAtlas.

Reply:
Rather than potentially regress something here, I think it would be better to investigate this before merging.

Reply:
Done! It touches several files which is why I wanted to wait for the next PR, but it's just one small commit so I think it's fine.